src/tx-fees.js

```diff
@@ -68,7 +70,35 @@ async function estimateWithdrawGasLimit (token, merkleSiblingsLength, amount, ov
   }
 }
 
+/**
+ * Estimates the gas limit for a withdrawCircuit
+ * @param {Object} token - The token information object as returned from the API
+ * @param {BigInt} amount - The amount to be withdrawn
+ * @param {Object} overrides - Transaction overrides
+ * @param {Object} signerData - Signer data used to send the transaction.
+ * @param {String} providerUrl - Network url (i.e, http://localhost:8545). Optional
+ * @param {Boolean} isInstant - Whether it should be an Instant Withdrawal
```

Comment:
this shouldn't bee the last parameter. signerData and providerUrl should be last as they are optional

Reply:
I understand that isInstant is Optional as well because it has a value by default